README.md

```diff
@@ -1,16 +1,52 @@
 # Open Hospital
 
-This repository is used to assemble the portable (or all-in-one) packages of the [Open Hospital][openhospital] project, which you can download [here][releases].
+[Open Hospital][openhospital] (OH) is an electronic health record (EHR) application.
+Open Hospital is currently provided as a desktop application written in Java
+and it's designed to work without an internet connection.
+
+Open Hospital is composed by the following components, hosted in separated repositories:
+ - [OH Core][core], a library that contains the business logic and the data abstraction layer
+ - [OH GUI][gui], which provides a graphical user interface (GUI) made with Java Swing
+ - [OH Doc][doc], which contains the user and admin documentation in Asciidoc format
+ - [OH UI][ui], a web user interface that consists of a React SPA (single page application) [*WIP*]
+ - [OH API][api], a web server that exposes REST APIs over the Core component, and it's used by the UI component [*WIP*]. 
+
+This repository is used to assemble the portable (or all-in-one) packages of Open Hospital, which you can download [here][releases].
 
 ## How to contribute
 
-If you'd like to contribute to the Open Hospital project, please read [CONTRIBUTING.md][contributing].
+There are several ways in which you can contribute to Open Hospital:
+
+- try the [desktop application][releases] or the early versions of the [web UI][ui]
+- request new features or report issues on [JIRA][jira] ([here][good-first]'s a list of *good-first-issues*)
+- improve the [documentation][doc]
+- contribute code patches to one of the components
+
+> NB: GitHub Issues are disabled on the repositories of individual components. 
+They are only enabled on this repository, with the purpose of collecting reports of bugs and 
+issues about this repository only. Instead, Jira is used to collect issues and plan 
+the development across all OH components.
```

Comment:
and also for end-users? Jira Service Desk is disabled at the moment, no plans to enable it

Reply:
I'm a bit confused about the whole role of github issues vs jira issues, so I'll let you fix the phrasing as you prefer